source/examples/xgboost-azure-mnmg-daskcloudprovider/notebook.ipynb

```diff
@@ -0,0 +1,2563 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    score = mean_squared_error(y_pred, _y_test)
</pre><p> think there's a problem when loading the locally saved model to the remote dask workers, but i haven't been able to figure out how to resolve -- the only thing blocking this PR from merging. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='87'/>